esrally/mechanic/telemetry.py

```diff
@@ -712,65 +712,6 @@ def attach_to_node(self, node):
         self.metrics_store.put_value_node_level(node.node_name, "node_startup_time", self.timer.total_time(), "s")
 
 
-class MergeParts(InternalTelemetryDevice):
-    """
-    Gathers merge parts time statistics. Note that you need to run a track setup which logs these data.
-    """
-    MERGE_TIME_LINE = re.compile(r": (\d+) msec to merge ([a-z ]+) \[(\d+) docs\]")
```

Comment:
This results in (at least) one new unused import `re` in this module. Can you please check unused imports and remove them?